lib/Elastica/QueryBuilder/Version/Latest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace Elastica\QueryBuilder\Version;
+
+/**
+ * Latest elasticsearch DSL.
+ *
+ * @link https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html
```

Comment:
@webdevsHub Here I would add a comment: Latest always refer to the version mentioned in README.md

Like this it never has to be updated here :-)